src/platform/lumin-runtime/elements/properties/color-property.js

```diff
@@ -0,0 +1,34 @@
+import { PropertyDescriptor } from './property-descriptor.js';
+import chroma from 'chroma-js';
+
+// Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
+
+class ColorProperty extends PropertyDescriptor {
+  // Value is:
+  //   string: 'hotpink'; '#ff3399'; 'F39'
+  //   number: 0xff3399
+  //    array: [r, g, b, a?]
+  //   object:
+  //    { h:120, s:1, l:0.75 }
+  //    { l:80, c:25, h:200 }
+  //    { c:1, m:0.5, y:0, k:0.2}
+  //    { r:255, g:165, b:0, a: 0.8 }
+  validate (value) {
+    return ColorProperty.validate(value);
+  }
+
+  static validate (value) {
+    return chroma.valid(value);
```

Comment:
I assume that this just returns true or false.  If this is the case then we should do more than just validate here https://github.com/magic-script/magic-script-components-lumin/pull/67/files#diff-03b72f65ad12cfe07e6910640e273947R56 
I think maybe adding a method to return a sane default of validation fails is acceptable.